docs/tutorials/classification.ipynb

```diff
@@ -2,195 +2,433 @@
+   ]
   },
   {
    "cell_type": "markdown",
+   "metadata": {},
    "source": [
-    "3. Use `OneHotEncoder` to create an encoder, that will be used later to transform the training table.\n",
-    "* We use `OneHotEncoder` to transform non-numerical categorical values into numerical representations with values of zero or one. In this example we will transform the values of the sex column, hence they will be used in the model for predicting the surviving of passengers.\n",
-    "* Use the `fit` function of the `OneHotEncoder` to pass the table and the column names, that will be used as features to predict who will survive to the encoder.\n",
-    "* The names of the column before transformation need to be saved, because `OneHotEncoder` changes the names of the fitted `Column`s:\n"
-   ],
-   "metadata": {
-    "collapsed": false
-   }
+    "### Imputing columns `age` and `fare`\n",
+    "We fill in missing values in the `age` and `fare` columns with the mean of each column\n"
+   ]
   },
   {
    "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
    "source": [
-    "from safeds.data.tabular.transformation import OneHotEncoder\n",
+    "from safeds.data.tabular.transformation import SimpleImputer\n",
     "\n",
-    "encoder = OneHotEncoder(column_names=\"sex\").fit(train_table)"
-   ],
-   "metadata": {
-    "collapsed": false
-   },
-   "execution_count": null,
-   "outputs": []
+    "simple_transformer = SimpleImputer(column_names=[\"age\",\"fare\"],strategy=SimpleImputer.Strategy.mean())\n",
+    "_, transformed_raw_data = simple_transformer.fit_and_transform(raw_data)"
+   ]
   },
   {
    "cell_type": "markdown",
-   "source": [
-    "4. Transform the training table using the fitted encoder, and create a set with the new names of the fitted `Column`s:\n"
-   ],
    "metadata": {
     "collapsed": false
-   }
+   },
+   "source": [
+    "### Using `OneHotEncoder` to create an encoder and fit and transform the table\n",
+    "We use `OneHotEncoder` to transform categorical, non-numerical values into numerical representations with values of zero or one. In this example, we will transform the values of the `sex` column, so they can be used in the model to predict passenger survival.\n",
+    "- Use the `fit_and_transform` function of the `OneHotEncoder` to pass the table and the column names to be used as features for the prediction."
+   ]
   },
   {
    "cell_type": "code",
-   "source": "transformed_table = encoder.transform(train_table)",
+   "execution_count": 5,
    "metadata": {
     "collapsed": false
    },
-   "execution_count": null,
-   "outputs": []
+   "outputs": [],
+   "source": [
+    "from safeds.data.tabular.transformation import OneHotEncoder\n",
+    "\n",
+    "one_hot_encoder = OneHotEncoder(column_names=[\"sex\", \"port_embarked\"])\n",
+    "transformer_one_hot_encoder, transformed_raw_data = one_hot_encoder.fit_and_transform(transformed_raw_data)"
```

Comment:
```suggestion
    "fitted_one_hot_encoder, transformed_raw_data = one_hot_encoder.fit_and_transform(transformed_raw_data)"
```

Could also chain the call of `OneHotEncoder` and the call of `fit_and_transform`.